{u'description': [u"alunduil's personal overlay"],
 u'feeds': [u'https://github.com/alunduil/alunduil-overlay/commits/master.atom'],
 u'homepage': u'https://github.com/alunduil/alunduil-overlay',
 u'irc': None,
 u'name': u'alunduil',
 u'official': True,
 u'owner_email': u'alunduil@gentoo.org',
 u'owner_name': u'Alex Brandt',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'https://github.com/alunduil/alunduil-overlay.git',
               u'Git',
               u''),
              (u'git://github.com/alunduil/alunduil-overlay.git',
               u'Git',
               u''),
              (u'git@github.com:alunduil/alunduil-overlay.git', u'Git', u'')],
 u'status': u'official',
 u'supported': True}
$ pmaint sync alunduil
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
From https://github.com/alunduil/alunduil-overlay
   90fc0cf..ab0f9a5  master     -> origin/master
Updating 90fc0cf..ab0f9a5
Fast-forward
 dev-python/pyhamcrest/Manifest                |  4 ---
 dev-python/pyhamcrest/files/sphinx-rtd.patch  | 20 -----------
 dev-python/pyhamcrest/metadata.xml            | 10 ------
 dev-python/pyhamcrest/pyhamcrest-1.8.5.ebuild | 50 ---------------------------
 4 files changed, 84 deletions(-)
 delete mode 100644 dev-python/pyhamcrest/Manifest
 delete mode 100644 dev-python/pyhamcrest/files/sphinx-rtd.patch
 delete mode 100644 dev-python/pyhamcrest/metadata.xml
 delete mode 100644 dev-python/pyhamcrest/pyhamcrest-1.8.5.ebuild
*** syncing alunduil
*** synced alunduil
 * Sync succeeded
$ pmaint regen -t 2 alunduil
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
!!! !!! !!! !!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-emulation/docker-compose-1.3.3caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-portage/etest-9999

!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/aiohttp-0.16.6
!!! !!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/nose2-0.5.0
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/debtcollector-0.6.0
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: media-fonts/monoid-9999
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/os-client-config-1.4.0!!! 
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/oslo-i18n-2.1.0
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/oslo-config-2.0.0
!!! !!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-backup/holland-backup-xtrabackup-1.0.10
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/oslotest-1.9.0
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/oslo-utils-2.1.0
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/oslo-serialization-1.7.0
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/python-termstyle-0.1.10
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/supernova-2.0.3
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/docker-py-1.3.1
 * Cache regenerated successfully
